docs/slides/content/pair-and-mob-programming.md

```diff
@@ -127,21 +127,22 @@ computer.
 
 
 
-### A day of Mob Programming
+### A Day of Mob Programming
 <iframe width="1200" height="900" src="https://www.youtube.com/embed/dVqUcNKVbYg" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope" allowfullscreen></iframe>
 
 
 
 ##### Key Takeaways
 - **Mentoring of team members:** Sharing tips and tricks or shortcuts can widen both pairs' skill depth. This sharing can also bring junior associates up to speed very rapidly.
 - **Half the code:** Two minds working on the same problem makes for more efficiently written code, so less spaghetti.
-- **No pull requests:** code is being reviewed as it's written avoiding superficial reviews. They won't cut corners when pairing as someone is watching.
+- **No pull requests:** Code is reviewed as it's written, avoiding superficial reviews. The temptation to cut corners is reduced when pairing because someone is watching.
 - **Team bonding:** Pair and mob programming facilitates interactions. A room with mob and pair programming going on is louder. Happy coders lead to better code.
```

Comment:
```suggestion
- **Team bonding:** Pair and mob programming facilitates interactions. Instead of sitting in a room with headphones on, ignoring the world around, developers in pairs or mobs look engaged and happier. Happy coders lead to better code.
```

Reply:
Just tweaked one preposition. Otherwise, I like this suggestion!